src/main/java/com/google/sps/utils/data/DataCollection.java

```diff
@@ -134,6 +142,89 @@ private static void parseSpeciesTable(String url) throws IOException {
         }
     }
 
+    /**
+    * Scrapes EXTINCT_URL to get a list of all the URLs with information about species
+    * Page Structure:
+    *   <ul>
+    *     <li><a href="LINK"></a>[Text]</li>
+    *     <li><a href="LINK"></a>[Text]</li>
+    *   </ul>
+    *
+    * @return List of all the URLs to go to
+    */
+    private static List<String> parseExtinctList() throws IOException {
```

Comment:
Might be helpful to add a comment that explain that the links in this case are for the actual species page, rather than a page with a table.